Source/Core/VideoCommon/VertexLoader_Color.h

```diff
@@ -9,9 +9,38 @@
 enum class VertexComponentFormat;
 enum class ColorFormat;
 
+using Common::EnumMap;
+
+using VertexColorSizeTable =
+    EnumMap<EnumMap<u32, ColorFormat::RGBA8888>, VertexComponentFormat::Index16>;
```

Comment:
Since this is in a header, this will also affect anything that includes the header. Would it be possible to move both the using statements and the table initialization into the `VertexLoader_Color` class (and make them `private`)? Same with the other files.